fcrepo-configs/src/main/java/org/fcrepo/config/FedoraPropsConfig.java

```diff
@@ -131,6 +131,12 @@
     @Value("${fcrepo.cache.db.containment.timeout.minutes:10}")
     private long containmentCacheTimeout;
 
+    @Value("${fcrepo.cache.types.size.entries:1024}")
```

Comment:
Did these settings make it into the documentation?

Reply:
Not yet, I was waiting for the merge, but thanks for the reminder.

Reply:
It looks like not.  I'll add them.

Reply:
No worries.  I'll add them before I merge.

Reply:
Thank you!